src/api_generation/high_level.jl

```diff
@@ -40,6 +59,8 @@ function _generate_high_level_wrapper(
              using AWS.AWSServices: $service_name
              using AWS.Compat
              using AWS.UUIDs
+
+             MAPPING = $(aws_service_param_mapping)
```

Comment:
```suggestion
             const MAPPING = $(aws_service_param_mapping)
```
You don't expect to reassign `MAPPING` right?

Also I think it could use a better name. MAPPING just indicates its type and not its contents.

Reply:
I've added a comment describing the relationship and kept the same name. I feel like that's better than something more lengthy like `const JULIA_TO_AWS_SYNTAX_MAPPING`?

Reply:
`const PARAMETER_MAPPING` would be better I think. Just any word describing what is being mapped. Having a comment in addition is also good though.

Reply:
I like `SERVICE_PARAMETER_MAP`. It doesn't matter much if it's wordy